blockchain/blockchain.go

```diff
@@ -1581,6 +1581,12 @@ func (bc *BlockChain) writeBlockWithState(block *types.Block, receipts []*types.
 		return status, err
 	}
 
+	for _, module := range bc.executionModules {
```

Comment:
Instead of calling in the common call site writeBlockWithState, we might call it from worker.wait, as some other codes are already duplicated across insertchain - worker.wait

Reply:
Applied 32c8349